src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRequestController.java

```diff
@@ -35,7 +35,7 @@ public class AudioRequestController implements AudioRequestsApi {
         operationId = "addAudioRequest",
         summary = "Users can request audio for specific cases and date/time periods",
         description = "Adds a user audio request for processing",
-        tags = { "Audio Requests" },
+        tags = { "AudioRequests" },
```

Comment:
How come the space has been removed here?

Reply:
The OAS currently uses tags to group the operations. This was the gotcha I asked @cmagowan-hmcts about and he mentioned in the dev channel - https://hmcts-reform.slack.com/archives/C04T0CVJ28N/p1685530019602349